src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/behandling/BehandlingsstatistikkService.kt

```diff
@@ -155,14 +156,15 @@ class BehandlingsstatistikkService(
     private fun finnSaksbehandler(
         hendelse: Hendelse,
         gjeldendeSaksbehandler: String?,
-        behandlingId: UUID,
+        totrinnskontroll: Totrinnskontroll?,
     ): String {
         return when (hendelse) {
-            Hendelse.MOTTATT, Hendelse.PÅBEGYNT, Hendelse.VENTER, Hendelse.HENLAGT ->
-                gjeldendeSaksbehandler ?: error("Mangler saksbehandler for hendelse")
+            Hendelse.MOTTATT, Hendelse.PÅBEGYNT, Hendelse.VENTER ->
+                gjeldendeSaksbehandler ?: error("Mangler saksbehandler for hendelse=$hendelse")
 
             Hendelse.VEDTATT, Hendelse.BESLUTTET, Hendelse.FERDIG ->
-                totrinnskontrollService.hentSaksbehandlerSomSendteTilBeslutter(behandlingId)
+                totrinnskontroll?.saksbehandler ?: gjeldendeSaksbehandler
```

Comment:
Hmm. Er ikke `saksbehandler` og `beslutter` to ulike folk?

Reply:
Metoden er for å finne saksbehandler. 
Beslutter personen som har gjennomført totrinnskontrollen ja

Når man beslutter så ønsker man å finne saksbehandler som sendte til totrinnskontrollen

Reply:
Skjønner, takk!